### PR TITLE
subscription detail box changes and add status with subscription name…

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
… in service marketplace detail page

### CODE REVIEW

**Review by lavanya-bmw — APPROVED** [review]
lgtm

**Review by oyo — DISMISSED** [review]
okay - assuming that those noRefCheck functions will be implemented later

File: cx-portal/src/components/pages/ServiceMarketplaceDetail/components/MarketplaceSubscription/MarketplaceSubscriptionNames.tsx [hunk 1]
```diff
@@ -31,5 +32,24 @@ export default function MarketplaceSubscriptionNames({
...
-  return <span className="subscription-name">{data && data.offerName}</span>
+  return (
+    <div className="subscription-list">
+      {data && (
+        <span className="subscription-name">
+          <Typography variant="body2">{data.offerName}</Typography>
+          <Chip
+            color={data.status === 'ACTIVE' ? 'success' : 'info'}
+            label={data.status}
+            onClick={function noRefCheck() {}}
```
[the remark this answers is not in the document]
Comment (nidhigarg-bmw) [reply cx-portal/src/components/pages/ServiceMarketplaceDetail/components/MarketplaceSubscription/MarketplaceSubscriptionNames.tsx]: I removed them for now, Will add them later whenever need them

**Review by manojava-gk — APPROVED** [review]
lftm

**Pull request merged**